appveyor.yml

```diff
@@ -61,89 +61,58 @@ matrix:
     # verify without checkstyle (JDK11)
-    # Spotbugs check is disabled till https://github.com/spotbugs/spotbugs/issues/259
     - JAVA_HOME: C:\Program Files\Java\jdk11
       DESC: "verify without checkstyle (JDK11)"
-      CMD1: "mvn -e verify -Dcheckstyle.ant.skip=true -Dcheckstyle.skip=true -Dspotbugs.skip=true"
+      CMD: "./.ci/appveyor.bat verify_without_checkstyle_JDK11"
     # verify without checkstyle (JDK12)
-    # Spotbugs check is disabled till https://github.com/spotbugs/spotbugs/issues/259
     - JAVA_HOME: C:\Program Files\Java\jdk12
       DESC: "verify without checkstyle (JDK12)"
-      CMD1: "mvn -e verify -Dcheckstyle.ant.skip=true -Dcheckstyle.skip=true -Dspotbugs.skip=true"
-      CMD2: "   -Dtest=!FileContentsTest#testGetJavadocBefore,!FileTextTest#testFindLine*,"
-      CMD3: "!MainFrameModelPowerTest#testOpenFileWithUnknownParseMode,"
-      CMD4: "!TokenUtilTest#testTokenValueIncorrect2,"
-      CMD5: "!ImportControlLoaderPowerTest#testInputStreamThatFailsOnClose"
-      CMD6: "   -Djacoco.skip=true"
+      CMD: "./.ci/appveyor.bat verify_without_checkstyle_JDK12"
     # site, without verify (JDK8)
     - JAVA_HOME: C:\Program Files\Java\jdk1.8.0
       DESC: "site, without verify (JDK8)"
-      CMD1: "mvn -e -Pno-validations site"
+      CMD: "./.ci/appveyor.bat site_without_verify"
     # site, without verify (JDK9)
     - JAVA_HOME: C:\Program Files\Java\jdk9
       DESC: "site, without verify (JDK9)"
-      CMD1: "mvn -e -Pno-validations site"
+      CMD: "./.ci/appveyor.bat site_without_verify"
     # site, without verify (JDK10)
     - JAVA_HOME: C:\Program Files\Java\jdk10
       DESC: "site, without verify (JDK10)"
-      CMD1: "mvn -e -Pno-validations site"
+      CMD: "./.ci/appveyor.bat site_without_verify"
     # site, without verify (JDK11)
     - JAVA_HOME: C:\Program Files\Java\jdk11
       DESC: "site, without verify (JDK11)"
-      CMD1: "mvn -e -Pno-validations site"
+      CMD: "./.ci/appveyor.bat site_without_verify"
     # site, without verify (JDK12)
     - JAVA_HOME: C:\Program Files\Java\jdk12
       DESC: "site, without verify (JDK12)"
-      CMD1: "mvn -e -Pno-validations site"
-      CMD2: "   -Dtest=!FileContentsTest#testGetJavadocBefore,!FileTextTest#testFindLine*,"
-      CMD3: "!MainFrameModelPowerTest#testOpenFileWithUnknownParseMode,"
-      CMD4: "!TokenUtilTest#testTokenValueIncorrect2,"
-      CMD5: "!ImportControlLoaderPowerTest#testInputStreamThatFailsOnClose"
-      CMD6: "   -Djacoco.skip=true"
+      CMD: "./.ci/appveyor.bat site_without_verify_jdk12"
 
 build_script:
   - ps: >
-       (get-content env:CMD1)
-       + (get-content env:CMD2)
-       + (get-content env:CMD3)
-       + (get-content env:CMD4)
-       + (get-content env:CMD5)
-       + (get-content env:CMD6)
-       + (get-content env:CMD7)
-       + (get-content env:CMD8)
-       + (get-content env:CMD9)
-       + (get-content env:CMD10)
+       (get-content env:CMD)
```

Comment:
can we avoid cmd file creation and directly call required command ?
Something like this:
https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/invoke-expression?view=powershell-6

Reply:
Yes i will work on that too.